doc/howto/configuration.md

```diff
@@ -71,4 +84,9 @@ An example manifest entry may look like
     * `gen3-volume-location` the location where the user's API key file will be put into
     * `lifecycle-pre-stop` a string array as the container prestop command.
     * `lifecycle-post-start` a string array as the container poststart command.
-    * `friends` is a list of kubernetes containers to deploy alongside the main container and the sidecar in the kubernetes pod
+    * `friends` is a list of kubernetes containers to deploy alongside the main container and the sidecar in the kubernetes pod.
+    * `nextflow` is for configuration specific to Nextflow containers.
+      * `enabled` is false by default; if true, automatically create AWS resources required to run Nextflow workflows in AWS Batch - see the [Nextflow workspaces](/doc/explanation/nextflow.md) documentation for more details.
+      * `job-image-whitelist` are the only images that are allowed as Nextflow workflow containers. It supports wildcards `?` for a single character and `*` for multiple characters. Warning: setting the whitelist as an empty list allows all images!
```

Comment:
Would it be possible for us to block all containers in case of an empty list, and allow all containers if the list has an element `*`? This way, if someone wants to allow all containers, it would be a deliberate choice rather than an accidental one?

Reply:
I chose to interpret it as "all images are allowed" because:
- I am not sure how to differentiate `job-image-whitelist = []` (no images are allowed) from `job-image-whitelist` not being configured at all (all images are allowed), since Go automatically fills out missing fields with empty values - in the code i always get an empty list.
- This way the field is optional
- It wouldn't make sense to deploy a nextflow workspace and not allow any containers